Protos/V1/results_service.proto

```diff
@@ -19,4 +20,35 @@ service Results {
    * Get the id of the task that should produce the result
    */
   rpc GetOwnerTaskId(GetOwnerTaskIdRequest) returns (GetOwnerTaskIdResponse);
+
+  /**
+   * Create the metadata of multiple results at once
+   * Data have to be uploaded separately
+   */
+  rpc CreateResultsMetaData(CreateResultsMetaDataRequest) returns (CreateResultsMetaDataResponse) {}
+
+  /**
+   * Create one result with data included in the request
+   */
+  rpc CreateResults(CreateResultsRequest) returns (CreateResultsResponse) {}
+
+  /**
+   * Upload data for result with stream
+   */
+  rpc UploadResultData(stream UploadResultDataRequest) returns (UploadResultDataResponse) {}
```

Comment:
I thought you wanted to have an alternative call without stream for small results. Is it still something we would want?